ivc/tests/add.rs

```diff
@@ -103,9 +103,8 @@ pub fn interpreter_simple_add<
 ) {
     let a = env.read_column(AdditionColumn::A);
     let b = env.read_column(AdditionColumn::B);
-    env.hcopy(&(a.clone() + b.clone()), AdditionColumn::C);
     let c = env.read_column(AdditionColumn::C);
-    let eq = a.clone() + b.clone() - c;
+    let eq = a.clone() * a.clone() * b.clone() - c;
```

Comment:
File can be renamed. Having a higher degree constraint allows better test coverage, since quad columns come not just from IVC but also from APP. One can use this test to debug quad easier because IVC can be turned off, and then you have just 4 columns and 2-3 expressions.

It also allows to filter out potential bugs related to the higher degree -- initially expression evaluation was not working well for subtraction exactly because of the incorrect treatment of the minus operation, which would be hell to debug with (much more weighty) IVC constraints.

We can also switch this back to addition, but it would be (1) more work than renaming the test, and we would achieve (2) worse test coverage. So I don't see the motivation.